ui_linux_wayland_integration_dummy.cpp lacks destructor

https://github.com/desktop-app/lib_ui/blob/3f85ce53d68bae6ca090e3389dec035f1fae0db7/ui/platform/linux/ui_linux_wayland_integration_dummy.cpp lacks destructor

So linking of `tdesktop` using `-DDESKTOP_APP_DISABLE_WAYLAND_INTEGRATION=ON` fails with
```
ui_linux_wayland_integration_dummy.cpp:(.text+0x4f): undefined reference to `Ui::Platform::WaylandIntegration::~WaylandIntegration()'
```

--- a/ui/platform/linux/ui_linux_wayland_integration_dummy.cpp
+++ b/ui/platform/linux/ui_linux_wayland_integration_dummy.cpp
@@ -17,6 +17,8 @@ struct WaylandIntegration::Private {
 
 WaylandIntegration::WaylandIntegration() {
 }
+	
+WaylandIntegration::~WaylandIntegration() = default;
 
 WaylandIntegration *WaylandIntegration::Instance() {
 	if (!::Platform::IsWayland()) return nullptr;

